specification/configuration.md

```diff
@@ -163,6 +163,35 @@ Other requirements:
 - RUM library MUST use the Zipkin v2 JSON span exporter by default
 - RUM library MUST limit the number of sent spans to 100 in a 30 second window per `component` attribute value
 
+#### Serverless 
+
+Current serverless offering is composed of separate metrics (non OTel) and traces (OTel) solutions. 
```

Comment:
I suggest replacing "OTel" with "OpenTelemetry" and "spec" with "specification".

Reply:
Well.. "spec" is not used anywhere in the document  😆

Reply:
It's fine to use "OTel" throughout the text after using "OpenTelemetry".